python/lsst/ts/mtdomegui/model.py

```diff
@@ -62,11 +125,605 @@ def __init__(
 
         # TODO: Put this into the ts_config_mttcs in DM-48109.
```

Comment:
I think this TODO may be removed.

Reply:
Done in the commit: [17d71c3](https://github.com/lsst-ts/ts_mtdomegui/pull/4/commits/17d71c31a9d25a477c45cf52d78acfc78b39afce). Thanks!